src/Uno.UI.BindingHelper.Android/Uno/UI/UnoViewGroup.java

```diff
@@ -421,12 +424,17 @@ public boolean dispatchTouchEvent(MotionEvent e)
 			for (int i = getChildCount() - 1; i >= 0; i--) { // Inverse enumeration in order to prioritize controls that are on top
 				View child = getChildAt(i);
 
+				if (child.getVisibility() != View.VISIBLE)
```

Comment:
seems odd it is the only place in all this logic where we check visibility.

Reply:
Normally the visibility check would be handled by super.dispatch..., but this is the code path where we don't call it (because it doesn't account for the transform).